_installation_collection/DE/Wie_installiere_ich_den_metasfresh_Stack_mit_Docker.md

```diff
@@ -36,20 +36,23 @@ Diese Anleitung beschreibt die Installation der folgenden Dienste gemäß [diese
 
 ### metasfresh installieren
 
-1. Klone die Docker-Compose Datei.
+1. Klone die Docker-Compose Datei und wechsel in das neu erstelle Directory
 
-   `git clone https://github.com/metasfresh/metasfresh-docker.git`
+   ```
+   git clone https://github.com/metasfresh/metasfresh-docker.git
+   cd metasfresh-docker/
+   ```
 
 1. Kommentiere `http://beispiel.com:8080` aus und ersetze die Zeile durch einen vom Browser extern erreichbaren Hostnamen und Port.
 
-   `vi ./metasfresh-docker/docker-compose.yml`
+   `vi ./docker-compose.yml`
    ```
    ...
-  #environment:
+   #environment:
     #- WEBAPI_URL=http://beispiel.com:8080
    ...
    ```
-   >**Hinweis:** PORT `:8080` ist nur nötig, wenn nicht 80 verwendet wird.
+   >**Hinweis:** PORT `:8080` ist nur nötig, wenn ein anderer als port 80 verwendet werden soll. Siehe auch [Wie ändere ich die WebUI Ports für metasfresh-docker?](Wie_aendere_ich_die_ports_fuer_webui_in_docker.html)
```

Comment:
i'm not sure if this is the correct way of referencing. @metas-Kay can you tell?